dev/archery/archery/crossbow/cli.py

```diff
@@ -155,6 +155,49 @@ def submit(obj, tasks, groups, params, job_prefix, config_path, arrow_version,
         click.echo('Pushed job identifier is: `{}`'.format(job.branch))
 
 
+@crossbow.command()
+@click.option('--base-branch', '-b', default="master",
```

Comment:
It seems that we don't need short options such as `-b` because we run this from `02-source.sh`.